docs/REST API/rest-api.md

```diff
@@ -1084,6 +1087,7 @@ Modifies the Fleet's configuration with the supplied information.
 | webhook_url                          | string  | body  | _mdm.macos_migration settings_. The webhook url configured to receive requests to unenroll devices migrating from your old MDM solution. **Requires Fleet Premium license** |
 | custom_settings                   | list    | body  | _mdm.macos_settings settings_. Hosts that belong to no team and are enrolled into Fleet's MDM will have those custom profiles applied. |
 | enable_disk_encryption            | boolean | body  | _mdm.macos_settings settings_. Hosts that belong to no team and are enrolled into Fleet's MDM will have disk encryption enabled if set to true. **Requires Fleet Premium license** |
+| custom_settings                   | list    | body  | _mdm.windows_settings settings_. Hosts that belong to no team and are enrolled into Fleet's MDM will have those custom profiles applied. |
```

Comment:
Now I'm confused :D Shouldn't that be done in Marko's PR? Maybe that's an exception as something that was overlooked in his PR? I'm trying to get a feel for that new process where product takes care of the docs.

Reply:
I'm confused as well :D. I think it was an overlook. I sent a message to Marko and Noah [in Slack](https://fleetdm.slack.com/archives/C03C41L5YEL/p1699990267672439) about this.

Reply:
:+1: gotcha, I had missed that.